rocket/urls.py

```diff
@@ -1,13 +1,14 @@
 from django.conf.urls import url
 from . import views
 from django.contrib.auth.views import login,logout
-from .views import grievances,Profile
+from .views import grievances,Profile,UserProfileEdit
 
 urlpatterns=[
     url(r'^login/$', login, {'template_name':'rocket/login.html'}),
     url(r'^signup/$', views.signup.as_view(), name='signup'),
     url(r'^profile/$', Profile.as_view(), name='profile'),
     url(r'^edit_profile/$', views.edit_profile, name='edit_profile'),
+     url(r'^userprofile_edit/$',UserProfileEdit.as_view(), name='userprofile_edit' ),
```

Comment:
That's understandable from developer's perspective but as a user perspective this is not a good user experience. Have you encountered any website which has different profile update pages just because they have different models handling the data?